test/e2e_node/util_xfs_unsupported.go

```diff
@@ -18,6 +18,7 @@ limitations under the License.
 
 package e2enode
 
+//lint:ignore U1000 applied to non-linux build constraints.
```

Comment:
I'm not sure why this is required. For instance, the neighbor file has a definition for the `IsCgroup2UnifiedMode` that doesn't need the same ignore from linter.

Reply:
Yes, I found that this weird too. I'm not that familiar with the linter, but somehow without that line, the linter complains about `isXfs` not being used. The only difference with `IsCgroup2UnifiedMode` in the neighbor file is `isXfs` is not exported.

Reply:
It turns out to be an issue in my local.